code/go/0chain.net/smartcontract/minersc/dkg.go

```diff
@@ -224,8 +224,13 @@ func GetPhaseNode(statectx cstate.StateContextI) (
 
 func (msc *MinerSmartContract) setPhaseNode(balances cstate.StateContextI,
 	pn *PhaseNode, gn *GlobalNode, t *transaction.Transaction) error {
+
+	isViewChange, err := config.Configuration().ChainConfig.ReadValue("ViewChange")
+	if err != nil {
+		logging.Logger.Error("MinerSmartContract - setPhaseNode - Cannot read chain configuration", zap.Any("error", err))
+	}
 	// move phase condition
-	var movePhase = config.DevConfiguration.ViewChange &&
+	var movePhase = isViewChange == true &&
```

Comment:
`isViewChange` is enough, we don't need to do `isViewChange == true`.

Reply:
isViewChange is an interface. So we compare like this.